flyway/src/main/java/io/micronaut/flyway/FlywayConfigurationProperties.java

```diff
@@ -182,4 +187,24 @@ public boolean hasAlternativeDatabaseConfiguration() {
     public FluentConfiguration getFluentConfiguration() {
         return fluentConfiguration;
     }
+
+    /**
+     * Sets extra properties to be applied on configuration.
```

Comment:
The java doc from Flyway `FluentConfiguration` it says `Property names are documented in the flyway maven plugin`. @driverpt do you have link to those properties that we could add to the javadoc here?

Reply:
I think this is what you're looking for: https://documentation.red-gate.com/fd/parameters-184127474.html

Reply:
I have improved the javadoc https://github.com/micronaut-projects/micronaut-flyway/pull/450/commits/c475adc2e14a3bf986adbbaf880c82b7f4085cbf

Reply:
I have also added an example in the docs https://github.com/micronaut-projects/micronaut-flyway/pull/450/commits/1a5698863af2a824232b8981c1a6377a0b477b53